core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.ts

```diff
@@ -0,0 +1,58 @@
+import { of } from 'rxjs';
+
+import { Component, ElementRef, EventEmitter, Output, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+import { catchError } from 'rxjs/operators';
+
+import { AiContentService } from '../../shared/services/ai-content/ai-content.service';
+
+interface FormValues {
+    textPrompt: string;
+}
+
+@Component({
+    selector: 'dot-ai-content-prompt',
+    templateUrl: './ai-content-prompt.component.html',
+    styleUrls: ['./ai-content-prompt.component.scss']
+})
+export class AIContentPromptComponent {
+    @ViewChild('input') input: ElementRef;
+
+    @Output() formValues = new EventEmitter<FormValues>();
+    @Output() hide = new EventEmitter<boolean>();
+
+    loading = false;
+    form: FormGroup;
+
+    constructor(private fb: FormBuilder, private aiContentService: AiContentService) {
+        this.buildForm();
+    }
+
+    onSubmit() {
+        const textPrompt = this.form.value.textPrompt;
+
+        if (textPrompt) {
+            this.formValues.emit({ textPrompt });
+
+            this.hide.emit(true);
```

Comment:
```suggestion
            this.submit.emit(true);
```

Reply:
I renamed the output event from `hide` to `formSubmission` to address the eslint warnings related to using the reserved keyword 'submit'